core/kernel/dt_driver.c

```diff
@@ -266,6 +266,11 @@ void *dt_driver_device_from_node_idx_prop(const char *prop_name,
 	while (idx < len) {
 		idx32 = idx / sizeof(uint32_t);
 		phandle = fdt32_to_cpu(prop[idx32]);
+		if (phandle == 0) {
```

Comment:
While this style isn't rejected, please don't mix it with the other style in the same function.
So make this `if (!phandle)...`

Reply:
Indeed, I'll fixed that.